hooks/post-command

```diff
@@ -31,6 +33,17 @@ main() {
     exit 0
   fi
 
+  curl -s https://codecov.io/bash > codecov;
+
+  CODECOV_VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2);
+  CODECOV_SHA=$(sha1sum codecov)
+  REMOTE_SHA=$(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/$CODECOV_VERSION/SHA1SUM)
```

Comment:
see below for the raw endpoint rate limits. Also, this curl will need more resiliency than a single try.